### ISSUE TITLE
Another false offense for Style/CommentedKeyword 

### ISSUE THREAD

**Issue by marcandre** [bbatsov/rubocop#5259]
Currently complains about:

    defined?(SomeModule).should be_nil # Sanity check

`parser` is there exactly so there's no additional parsing to be done. See also #4886

```
$ rubocop -v
0.52.0
```

**Comment by gomo** [comment]
I have a similar problem too.

```rb
options = {
  ...
  end_buttons: true,        # Whether to display buttons to the first and last page.
  ...
}
```

```
$ rubocop -v
0.52.0
```
